modules/configuring-firewall.adoc

```diff
@@ -82,15 +82,57 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 |Cloud |URL | Port |Function
 
 |Alibaba
-|*.aliyuncs.com
+|`*.aliyuncs.com`
 |443, 80
 |Required to access Alibaba Cloud services and resources. Review the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/endpoints/endpoints_config.go?spm=a2c4g.11186623.0.0.47875873ciGnC8&file=endpoints_config.go[Alibaba endpoints_config.go file] to determine the exact endpoints to allow for the regions that you use.
 
-|AWS
+.11+|AWS
 |`*.amazonaws.com`
+
+Alternatively, if you choose to not use a wildcard for Amazon Web Services (AWS) APIs, you must allowlist the following URLs:
 |443, 80
 |Required to access AWS services and resources. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
 
+|`ec2.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|`events.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|`iam.amazonaws.com` [^1^]
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|`route53.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|`sts.amazonaws.com`
```

Comment:
@mjpytlak Since regional endpoints are default now ([CCO-257](https://issues.redhat.com/browse/CCO-257)), I suggest to add `sts.<aws_region>.amazonaws.com` to the list.

cc @patrickdillon

Reply:
@yunjiang29 Added `sts.<aws_region>.amazonaws.com`. Port 443 only or additional ports?

Reply:
@mjpytlak Yes, only 443 is required

Reply:
@mjpytlak other parts look good to me, thanks.